creator-node/src/index.ts

```diff
@@ -255,12 +255,12 @@ const startApp = async () => {
 
   await serviceRegistry.initServices()
   const nodeMode = config.get('devMode') ? 'Dev Mode' : 'Production Mode'
-  logger.info(`Initialized services (Node running in ${nodeMode})`)
+  logger.debug(`Initialized services (Node running in ${nodeMode})`)
 
   // eslint-disable-next-line @typescript-eslint/no-floating-promises
   serviceRegistry.initServicesAsynchronously()
   const appInfo = initializeApp(getPort(), serviceRegistry)
-  logger.info('Initialized app and server')
+  logger.debug('Initialized app and server')
   await serviceRegistry.initServicesThatRequireServer(appInfo.app)
```

Comment:
these are all potentially useful to see and they are so infrequent and tiny logs, not worth downgrading

would be good to focus efforts on frequent / large logs